NonResonant/README.md

```diff
@@ -0,0 +1,17 @@
+In "Hist2DSum_V0_SM_box.root" you find the distribution for both 
+     - the SM point (simulated from 100k events) - H0binX  
+     - sum of the events simulated to Mriond 2016 (SM + box + the V0 benchmarks from 2-13 (simulated from ~300 events/each) - H1binX  
```

Comment:
The bullet list shows broken as it is not following the [markdown syntax](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)
also: 
  * Mriond -> Moriond
  * 300 events -> 300k events